README.md

```diff
@@ -502,6 +511,91 @@ server:start()
 
 [socket_ref]: https://www.tarantool.io/en/doc/latest/reference/reference_lua/socket/#socket-tcp-server
 
+## Roles
+
+New roles could be accessed from this project.
+
+### `httpd` role
+
+This role can be used only with Tarantool 3. It allows configuring one or more
+HTTP servers. Those servers could be reused by several other roles.
+
+Example of the configuration:
+
+```yaml
+roles_cfg:
+  roles.httpd:
+    default:
+    - listen: 8081
+    additional:
+    - listen: '127.0.0.1:8082'
+```
+
+Server address should be provided either as a URI or as a single port
+(in this case, `0.0.0.0` address is used).
+
+User can access every working HTTP server from the configuration by name,
+using `get_server(name)` method.
+If the `name` argument is `nil`, the default server is returned
+(its name should be equal to constant `DEFAULT_SERVER_NAME`).
```

Comment:
And what is a value of this constant?

Reply:
`default` (this constant is also accessible from the role variable).

Reply:
Please, add it to the document. Configuration from the config should ideally be done without code.